source/MapPanel.cpp

```diff
@@ -63,10 +63,10 @@ namespace {
 	//  - first        -> ships
 	//  - first.first  -> ships (in-flight)
 	//  - first.second -> ships (parked)
-	//  - second       -> outfits (in storage)
+	//  - second       -> map of planets to number of outfits on that planet
 	void TallyEscortsAndOutfits(const vector<shared_ptr<Ship>> &escorts,
 		const std::map<const Planet *, CargoHold> &outfits,
-		map<const System *, pair<pair<int, int>, int>> &locations)
+		map<const System *, pair<pair<int, int>, map<const Planet *, unsigned>>> &locations)
```

Comment:
I think it is possibly more useful to display a tool-tip in the orbits scene, when the player hovers over the individual planet, rather than further expand the system-level tool-tip with planet-specific information.

Reply:
Gross

Reply:
@tehhowch The orbits scene also shows unlandable planets, and isn't currently used for anything, so players might not realize it has a tooltip with useful information. In contrast, the system's tooltip is already used, and describes a clickable thing that even has graphical annotations that indicate when the tooltip will contain specific information. So I think putting the tooltip on the system is more useful than putting it on the orbits.

Reply:
@MasterOfGrey What is and why?

Reply:
I like tehhowch idea. The argument that the orbits scene is not actively used is actually a good argument to start using it for more information like this.
We could also put a special marking on planets in the orbits scene that have stored items to make them more explicit/discoverable; maybe the same marker as [on systems](https://github.com/endless-sky/endless-sky/pull/4443#issuecomment-583619017)?

Reply:
For systems I would keep the total number of outfits stored in the pop-up, or maybe a listing of stored outfits per type (and then use tehhowch idea to give the per planet information).

Reply:
> The argument that the orbits scene is not actively used is actually a good argument to start using it for more information like this.

:thinking:

For discoverability, perhaps we could update [the help.txt "UI tutorial" messages](https://github.com/endless-sky/endless-sky/blob/13d22b84a8bd01070ab2705144511c7c69129c1f/data/help.txt#L69-L82) to point out the existence of orbits scene tooltips.

>  We could also put a special marking on planets in the orbits scene that have stored items to make them more explicit/discoverable;

Wouldn't this require a few more additions to the orbits scene, though? For starters, adding something to distinguish landable objects from other objects?

> maybe the same marker as [on systems](https://github.com/endless-sky/endless-sky/pull/4443#issuecomment-583619017)?

Should the "This system is a mission destination/stopover" markers also be applied to individual planets in the orbits scene, then? Especially in case you have missions to, say, both Luna and Earth? (And for that matter, it might be nice to have a subtle indication in the _system's_ mission destination markers that the destinations are not all on the same planet, but that's getting off-topic.)

Sorry if I sound like I have a mild case of featuritis. I'm just trying to run with @tehhowch's idea and see where it leads. If I had featuritis, I'd be proposing to use the orbits scene to indicate the government of each planet (#3768) :) 

> For systems I would keep the total number of outfits stored in the pop-up, or maybe a listing of stored outfits per type (and then use tehhowch idea to give the per planet information).

How about just the number of outfits and their total market value? Though there's a small can of worms here regarding _which_ value to show: the "As new" price, the depreciated-as-of-today sale price, or the depreciated-as-of-reaching-that-system sale price.

Reply:
The habitable (~= landable) planets already seem to stand-out with a blue color in the orbits scene.
Making the landable objects in the orbits scene match the "legend" selection (like showing friendly/hostile and showing governments when selected) sounds nice, but for a separate PR.
Adding the "mission destination/stopover" markers also to the orbit scene also sounds nice, but also for a separate PR.
Adding some tooltips could also be nice.

I would expect that storing outfits would typically be done to re-use them later. So when looking for stored outfits, then the total number and the exact types are interesting to a player. The value of the outfit(s) does not sound like something useful to display for stored outfits (especially since you can already lookup the "new price" value in the outfitter screen.)

Reply:
Mostly agreed, just one thing:

> I would expect that storing outfits would typically be done to re-use them later. So when looking for stored outfits, then the total number and the exact types are interesting to a player. The value of the outfit(s) does not sound like something useful to display for stored outfits (especially since you can already lookup the "new price" value in the outfitter screen.)

I proposed the number and total value of outfits as the tooltip for the system; just a summary so the tooltip doesn't get too crowded. The tooltip of the planet could give the exact list of outfits, even broken out by category (same categories as in the outfitter screen). Though it might be better to instead add a filter to the map outfitter view that shows stored outfits (like #5941 does, but without showing outfits for sale in the list at the same time; i.e., a filter that shows _only_ semidim items, rather than both semidim and dim items).

Reply:
Sounds good, but I still feel that there is little value in showing the outfits total value here. Showing the total amount per planet (as done earlier in this thread) per system feels more valuable to me and there will usually not be more than 2 outfitters in a system anyway, so the risk of overcrowding also seems small.

Reply:
OK, so let's the keep the system tooltip as it stands, then. I'll look into implementing a per-planet tooltip, unless y'all prefer to merge the systems tooltip first and do the planets tooltip in a follow-up PR. (That might make sense if there's a release around the corner.)

Reply:
@jostephd sorry, my comment had been at tehhowch's suggestion of a tool-tip in the orbits scene

Reply:
> I think it is possibly more useful to display a tool-tip in the orbits scene, when the player hovers over the individual planet, rather than further expand the system-level tool-tip with planet-specific information.

Looking into this. The system tooltips are handled by MapPanel::Hover and MapPanel::DrawTooltips. The former is already marked `override` to enable MapSalesPanel::Hover. I guess I should add a MapDetailPanel::Hover, then. What should it do if the x,y coordinates are outside the orbits scene? Should it just `return false` so UI::Handle can loop on, or should it call MapPanel::Hover as MapSalesPanel::Hover does?